### DISCUSSION TITLE
Search in Settings - Positioning

### DISCUSSION THREAD

**Discussion by DolomitesTom** [devemux86/cruiser#70] [Q&A] [answered]
The search function in settings is useful. But it always jumps to the top of the list and not to the selected setting. Can this be a bit more precise by jumping directly to the selected option?

**Comment by devemux86** [comment]
Thanks for the suggestion.

I tested it, but it did not work reliably, so I want to check it again. :slightly_smiling_face:

**Comment by devemux86** [comment]
> Can this be a bit more precise by jumping directly to the selected option?

It will work like this in the next Cruiser update.

It will display its parent screen like now and scroll to the option (if off screen). Is it what you thought?
(it will not open the option's dialog, as it would not be convenient and only some options have dialog)

**Reply by DolomitesTom** [reply]
Yes, that fits, thank you

**Comment by devemux86** [comment, marked as answer]
Implemented in [Cruiser 3.0.13](https://play.google.com/store/apps/details?id=gr.talent.cruiser).

**Reply by DolomitesTom** [reply]
works fine
